mixer/cmd/build.go

```diff
@@ -604,6 +609,8 @@ func runBuildDeltaPacks(cmd *cobra.Command, args []string) error {
 		fail(err)
 	}
 	setWorkers(b)
+	b.IgnoreMissing = buildFlags.ignoreMissing
+
```

Comment:
This is the only place that this value needs to be set. Delta packs are not generated by the other build commands, including build all.